rfcs/064-graph-data-model/README.md

```diff
@@ -0,0 +1,129 @@
+
+| Concept type  | Count |
+| ------------- | ----- |
+| Person    | 364483    |
+| Concept   | 122812    |
+| Organisation  | 57517 |
+| Place | 11149 |
+| Agent | 8828 |
+| Meeting   | 5153 |
+| Genre | 1494 |
+| Period    | 1032 |
+
+In some cases, these manually tagged concepts already come with an identifier from an external ontology (see table below). These sources are also added to concept nodes as a property.
+
+| Concept source    | Count |
+| --------  | ------    |
+| label-derived | 308908 |
+| lc-names  | 197509 |
+| lc-subjects   | 37343 |
+| nlm-mesh  | 28425 |
+| viaf  | 153 |
+| fihrist   | 130 |
+
+### SourceConcept
+
+Nodes for concepts from the following external ontologies: LCSH, Wikidata, or MeSH. Aside from the source identifier and label, properties include a description, alternative identifiers , and alternative labels/synonyms (see [sourceconcept.yaml](yaml_files/sourceconcept.yaml)). Additional properties can be added to incorporate more information from source ontologies, if needed. A full list of source properties is available for [MeSH](https://www.nlm.nih.gov/mesh/xml_data_elements.html) and [Wikidata](https://www.wikidata.org/wiki/Wikidata:Database_reports/List_of_properties/all). It is worth noting that Wikidata in particular has a wide range of properties, including links to various other external databases (e.g.the National Portrait Gallery, OpenAlex and many more), and a decision may need to be made whether to include all of these under `alternative_ids`.
+
+### SourceName
+
+Nodes for names from LCNAF (excluding concepts which are an instance of `MADS/RDF Geographic`, see below) and corresponding Wikidata concepts. MeSH is not included here as it does not include any names. Additional properties are included on `SourceName` nodes which specifically provide relevant context for names (see [sourcename.yaml](yaml_files/sourcename.yaml)): date of birth, date of death, and place of birth. Note that, for simplicity, there is currently no further split into different name categories such as organisation or meeting names (therefore the additional birth date/place properties for these will be empty). However, it should be relatively straightforward to split these further, if necessary, as they are identifiable via source vocabulary subdivisions and/or parent terms. For instance, there may be a need to incorporate specific information for organisations such as their location, founder etc.
+
+### SourceLocation
+
+Nodes for concepts from LCSH or LCNAF which are an instance of `MADS/RDF Geographic` and MeSH Geographicals (`Z` tree codes). Location-specific properties are included on `SourceLocation` nodes (see [sourcelocation.yaml](yaml_files/sourcelocation.yaml)), such as their coordinates.
```

Comment:
Can we expand acronyms the first time we use them or add a glossary to make this a little more accessible to readers not familiar, (I had to Google [LCNAF](https://id.loc.gov/authorities/names.html)).